core/templates/domain/question/question-update.service.ts

```diff
@@ -77,7 +77,9 @@ export class QuestionUpdateService {
     return changeDict[paramName];
   }
 
-  _getNewPropertyValueFromChangeDict(changeDict: BackendChangeObject): string {
+  _getNewPropertyValueFromChangeDict(
+      changeDict: BackendChangeObject
+  ): string | string[] {
```

Comment:
The return type of `_getParameterFromChangeDict()` is `string`, inconsistent with the return type of this function.

Reply:
Good catch. Updated, thanks.